docs/static_site/src/assets/js/jquery-3.3.1.min.js

```diff
@@ -0,0 +1,20 @@
+/*!
```

Comment:
Is keeping this in GitHub the best course of action? Couldn't we store this in an S3 bucket and pull it during build time via Jenkins?

Reply:
Good idea, do we have existing S3 for this purpose?

Reply:
Could use the `mxnet-website-static-artifacts` bucket for now, or create a new bucket.

Reply:
Thought this through again and found it could import potential problems by storing it on S3. It will require pulling the script when not only build on Jenkins, but also develop locally. If someone has trouble or high latency accessing our S3, the building process could break.
Found a more general approach would be using node package manager. It helps a lot when managing many dependencies, however we only have this one dependency now, it would import more complexity than benefit.
I would say for now still keep it within the project, ([other projects](https://github.com/apache/incubator-echarts-website/blob/asf-site/en/vendors/jquery/jquery.js) also do), and keep NPM as a candidate, switch to it when we need to handle more dependencies :)

Reply:
How about linking to the jquery cdn? https://code.jquery.com/

Reply:
This is the current way discussed in #18367, was to replace google cdn when hosting locally was not a solution. After Apache enabling Chinese CDN, the server is pretty fast and stable. So switch to host the scripts locally.

Reply:
Regardless of where the website developer is located geographically, they will always be using Jenkins to build the website - so local latency to S3 isn’t a concern. Since we’re not anticipating changing this file, it seems unnecessary to include it in version control systems and not in S3.

Reply:
Agreed that NPM would introduce more complexity than is necessary for including this single file, but pulling the file from S3 at build time would be as simple as adding a `wget` call to the Makefile for building the static site - see [here](https://github.com/apache/incubator-mxnet/pull/18340/files#diff-8938828e17dfd3968ac4048a65029ecfR23) for an example.

Reply:
Agreed on the point of building on Jenkins. However, the problem (local latency) I was concerning is that when develop the main site locally (run `jekyll serve` and website host on 127.0.0.1), it is also the time that the script need to be pulled from S3, and geographical location might affect the accessibility to this script.
If we really want to get rid of JQuery on Git, a way I can think of is that on one hand using S3/CDN for building on Jenkins, and for local dev, updating the doc/makefile to add the wget JQuery command before `jekyll serve`. wdyt?